README.md

```diff
@@ -268,31 +304,31 @@ export INPUT_GITHUB_TOKEN=$(printenv <your-env-token-var>)
 python3 ./<path-to-action-project-root>/main.py
 ```
 
-### Contribution Guidelines
+## Contribution Guidelines
 
 We welcome contributions to the Generate Release Notes Action! Whether you're fixing bugs, improving documentation, or proposing new features, your help is appreciated.
 
-#### How to Contribute
+### How to Contribute
 - **Submit Pull Requests**: Feel free to fork the repository, make changes, and submit a pull request. Please ensure your code adheres to the existing style and all tests pass.
 - **Report Issues**: If you encounter any bugs or issues, please report them via the repository's [Issues page](https://github.com/AbsaOSS/generate-release-notes/issues).
 - **Suggest Enhancements**: Have ideas on how to make this action better? Open an issue to suggest enhancements.
 
 Before contributing, please review our [contribution guidelines](https://github.com/AbsaOSS/generate-release-notes/blob/master/CONTRIBUTING.md) for more detailed information.
 
-### License Information
+## License Information
```

Comment:
Look at the all 2nd level headlines and make sure, that you have them in the TOC.

Reply:
Chapters structure should be fixed same as TOC.